packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -850,6 +850,36 @@ const replaceNodeHtmlLinks = ({ wpUrl, nodeString, node }) => {
   return nodeString
 }
 
+// replaces specific string or regex with a given string from the plugin options config
+const replaceNodeHtmlStrings = ({ nodeString, node, pluginOptions }) => {
```

Comment:
Can we rename this to `findAndReplaceNodeStrings`? This is replacing on stringified JSON not only on HTML.

Reply:
Renamed to `searchAndReplaceNodeStrings`